extensions/resteasy-reactive/rest-client-reactive/runtime/pom.xml

```diff
@@ -18,6 +18,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jaxrs-client-reactive</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.smallrye.stork</groupId>
+            <artifactId>smallrye-stork-microprofile</artifactId>
```

Comment:
why stork-microprofile?

Reply:
microprofile config support for Stork. In the future, it may also bring in some CDI support or sth

Reply:
I would state `config` then.

Reply:
will do, I need another relase of Stork for this. If you want to review the change in Stork: https://github.com/smallrye/smallrye-stork/pull/77 :)

Reply:
done